### PR TITLE
Create profile for aws credentials

### PR DESCRIPTION
**Pull request by swidnow2**

### CODE REVIEW

File: src/test/kotlin/com/atlassian/performance/tools/hardware/ApplicationScale.kt [hunk 1]
```diff
@@ -32,7 +32,7 @@ class ApplicationScales {
                 .virtualUsers(150)
                 .ramp(Duration.ofSeconds(90))
                 .flat(Duration.ofMinutes(20))
-                .maxOverallLoad(TemporalRate(30.0, Duration.ofSeconds(1)))
+                .maxOverallLoad(TemporalRate(1000.0, Duration.ofSeconds(1)))
```
Comment (dagguh) [inline src/test/kotlin/com/atlassian/performance/tools/hardware/ApplicationScale.kt]: This is out of scope

**Review by dagguh — CHANGES_REQUESTED** [review]

File: src/test/kotlin/com/atlassian/performance/tools/hardware/ApplicationScale.kt (hunk 1 above)
Comment (swidnow2) [reply src/test/kotlin/com/atlassian/performance/tools/hardware/ApplicationScale.kt]: Yes, I fixed it.

**Review by dagguh — APPROVED** [review]
